tables/automl/endpoint_test.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+import automl_tables_set_endpoint
+
+PROJECT = os.environ["GCLOUD_PROJECT"]
+
+
+def test_client_creation(capsys):
+    client = automl_tables_set_endpoint.create_client_with_endpoint(PROJECT)
+    print(client.list_datasets())
```

Comment:
Would it be possible to have the `create_client_with_endpoint` method print something, and then assert on your print? Then, you would be testing your sample, rather than the actual AutoML underlying function. 

Also, if testing is something you're not 100% comfortable with, do feel free to put some time on my calendar and we can take 20-30 min to have a faster back and forth about what you think is most appropriate for this sample :)

Reply:
+1 to @leahecole 's suggestion.

For the set endpoint kind of samples, we typically use a really simple API call to verify that the client was created and operates correctly.

See example [here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/vision/cloud-client/detect/set_endpoint.py).

I suggest copying the output of the regular Tables [list_datasets sample](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/tables/automl/automl_tables_dataset.py#L59).

Reply:
+1 to just executing `list_datasets` since the goal is to check that the client was constructed correctly.